common/static/karma_common.conf.js

```diff
@@ -69,6 +70,6 @@ var options = {
     ]
 };
 
-module.exports = function (config) {
+module.exports = function(config) {
```

Comment:
@andy-armstrong @cahrens Let's chat about it here: https://github.com/bjacobel/eslint-config-edx/pull/1#issuecomment-227159404